gui/wxpython/gui_core/toolbars.py

```diff
@@ -175,11 +175,10 @@ def EnableLongHelp(self, enable):
                     continue
                 else:
                     internal_label = tool[0][0]
+            elif tool[0] == "":  # separator
```

Comment:
Lines 176 and 177 could be reduced to:
```
                internal_label = tool[0][0]
```
given the `continue` above...

Reply:
I think that's another rule that will be fixed on its own https://docs.astral.sh/ruff/rules/superfluous-else-continue/ (RET503)